libs/network/include/network/p2pservice/p2p_http_interface.hpp

```diff
@@ -156,7 +156,7 @@ class P2PHttpInterface : public http::HTTPModule
       block["hash"]         = "0x" + b->hash.ToHex();
       block["previousHash"] = "0x" + b->previous_hash.ToHex();
       block["merkleHash"]   = "0x" + b->merkle_hash.ToHex();
-      block["proof"]        = "0x" + b->proof.header().ToHex();
+      block["proof"]        = "0x" + b->merkle_hash.ToHex();  // TODO(HUT): check non breaking here.
```

Comment:
@ejfitzgerald is this going to break block explorer?

Reply:
(if I remove this field that is)

Reply:
I guess I could switch it to entropy hex instead

Reply:
Just remove it, it is a simple update to block explorer anyway. Entropy hash makes more sense anyway

Reply:
OK, I will remove this completely